assets/css/admin-custom.css

```diff
@@ -117,3 +117,54 @@ div.question_boolean_fields { margin-bottom: 10px; }
 .sensei-dismissible-link {
     text-decoration: none;
 }
+
+/**
+ * Sensei custom navigation
+ */
+.sensei-custom-navigation {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: flex-start;
+	color: #000000;
+	margin-bottom: 20px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__title h1 {
+	color: #1E1E1E;
+	font-size: 20px;
+	margin-right: 25px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: center;
+	justify-content: flex-start;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar a {
+	margin-left: 0;
+	margin-right: 20px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar a.page-title-action {
+	top: auto;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar a.sensei-custom-navigation__navbar__navtab {
+	margin-right: 25px;
+	color: #1E1E1E;
+	text-decoration: none;
+	font-size: 14px;
+	padding-bottom:  10px;
+	border-bottom: 4px solid transparent;
+}
```

Comment:
Hmm, the navigation looks a bit too close to the top bar. Do you think it would be better to align the text with the page title?

Before:
<img width="703" alt="Screenshot on 2022-01-24 at 16-04-56 (1)" src="https://user-images.githubusercontent.com/1612178/150797582-66aecc7d-a073-4d3f-b204-303bc9365e03.png">

After (just added `margin-top: 15px` to the navtabs, but I'm not sure if this is the correct way):
<img width="707" alt="Screenshot on 2022-01-24 at 16-08-20" src="https://user-images.githubusercontent.com/1612178/150797919-eebac083-501b-47b8-90ba-7a19073f163e.png">

Reply:
It is not aligned with h1 in the design :) And it is questionable. Aligned approach looks better for me too.
But yes, there is too little space, anyway.

Reply:
Yeah, I'm not totally sure which would be better here. If we align the tabs, they don't look like tabs that much, so some users could be confused.

@donnapep @LavaGolem what do you think?

Reply:
Updated with the layout change: https://github.com/Automattic/sensei/pull/4672/commits/7e32384a664b0e7a92da10bdf7d45b98d128f026

Reply:
So I have a couple of questions here: 
1. Why isn't it in two rows? 
2. I think it definitely should be aligned since there are 3 different lines at the moment: Title, links, buttons, bars

Reply:
So the issue here is the padding on the title

```.wrap > h2:first-child, .wrap [class$="icon32"] + h2, .postbox .inside h2, .wrap h1```

Since it is tricky to remove it from header we should add it to these tabs also: 

```padding: 9px 0 4px 0```

Reply:
@merkushin  **align-items** will not fix it, since it is already good :D it is set to ```align-items: center``` which is good, but the padding on the **Course** header is making it unaligned

Reply:
Yes, you're right, thanks! I'll fix it.

Reply:
I removed padding from h1.
<img width="367" alt="CleanShot 2022-01-25 at 18 18 39@2x" src="https://user-images.githubusercontent.com/329356/151004785-e34afe89-d007-4bb8-82e9-84bd2bc1d845.png">